.github/workflows/main.yml

```
@@ -49,3 +49,11 @@ jobs:
         run: |
           if [[ -d ./torch-extensions ]]; then rm -rf ./torch-extensions; fi
           TORCH_EXTENSIONS_DIR=./torch-extensions pytest --durations=0 --forked --verbose tests/unit/
+
+      - name: Transformer tests
+        run: |
+          if [[ -d ./torch-extensions ]]; then rm -rf ./torch-extensions; fi
+          git clone https://github.com/huggingface/transformers
+          cd transformers
+          pip install -e .[testing]
```

```suggestion
          pip install -e .
          pip install sacrebleu rouge-score nltk parameterized psutil datasets
```

As `faiss-cpu` won't build, let's hardcode just the packages we want for now.